docs/sphinx/using/cudaq/kernel.rst

```diff
@@ -23,22 +23,22 @@ CUDA Quantum follows a similar pattern. Specifically, in an effort to better ena
 the development of **generic** libraries of quantum algorithmic primitives
 and applications, CUDA Quantum defines quantum device code as stand-alone typed
 callables in C++. A typed callable in C++ is any user-defined :code:`struct`
-or :code:`class` that provides an :code:`void operator()(Args...) {}`
-overload (an operator-call overload). Implicitly typed callables - C++
-lambdas, which compiler implementations enable as automated type definitions
-- also provide this typed-callable pattern. CUDA Quantum requires that these callable
-definitions be annotated in some way to indicate that this expression is meant
-for compilation and execution on the quantum device.
+or :code:`class` that provides an operator-call overload
+(:code:`void operator()(Args...) {}`). 
+C++ lambdas |---|  enabled as automated type definitions through compiler implementations
+|---| also implicitly provide this typed-callable pattern. CUDA Quantum requires 
+that these callable definitions be annotated in some way to indicate that this 
+expression is meant for compilation and execution on the quantum device. 
 
-CUDA Quantum distinguishes between two separate kinds of kernel expressions - entry-point
-and pure-device quantum kernels. Entry-point kernels are those that can be
+CUDA Quantum distinguishes between two separate kinds of kernel expressions: entry-point 
+and pure-device quantum kernels. Entry-point kernels are those that can be 
 called from host code, while pure-device kernels are those that can only be
-called from other quantum kernel code. See the `specification
-<https://nvidia.github.io/cuda-quantum/latest/specification/cudaq/kernels.html#quantum-kernels>`__
+called from other quantum kernel code. See the :doc:`specification <specification/cudaq/kernels>`
 for more detail,  but here we note that the "typed" requirement can be relaxed
 for pure-device kernels:
 
 .. code-block:: cpp 
+.. |---| replace:: —
```

Comment:
This location is not a good place since it interferes with the code-block here. Please add it before the code-block directive and surrounded by empty lines.